src/c/core/session/stream/output_reliable_stream.c

```diff
@@ -158,9 +158,10 @@ bool uxr_update_output_stream_heartbeat_timestamp(uxrOutputReliableStream* strea
         }
         else if(current_timestamp >= stream->next_heartbeat_timestamp)
         {
-            int64_t increment = MIN_HEARTBEAT_TIME_INTERVAL << (++stream->next_heartbeat_tries % MAX_HEARTBEAT_TRIES);
+            int64_t increment = MIN_HEARTBEAT_TIME_INTERVAL << (stream->next_heartbeat_tries % MAX_HEARTBEAT_TRIES);
             int64_t difference = current_timestamp - stream->next_heartbeat_timestamp;
             stream->next_heartbeat_timestamp += (difference > increment) ? difference : increment;
+            stream->next_heartbeat_tries++;
```

Comment:
I like the post-increment in these situations because the code stays more alignment among the different lines. The ASM generated by the compiler is the same in both situations.